### PR TITLE
added slider

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5500_Documentation_slider_slider html

### CODE REVIEW

File: Documentation/button/button.html [hunk 1]
```diff
@@ -148,5 +136,28 @@ <h2>Floating Icon Button</h2>
...
+              ><i class="fa-brands fa-github" style="font-size: 40px"></i
+            ></a>
+          </li>
+
+          <li class="list-inline">
+            <a
+              class="links"
+              target="_blank"
+              href="https://www.linkedin.com/in/priyanshi-tiwari-90594a221/"
+              ><i class="fa-brands fa-linkedin-in" style="font-size: 40px"></i>
```
Comment (Tanjum786) [inline Documentation/button/button.html:156]: avoid in-line styling

File: Documentation/slider/slider.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <!-- <link rel="stylesheet" href="../../index.css" /> -->
+    <link rel="stylesheet" href="../../Stylesheets/main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/6026902e47.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Stark Buttons</title>
```
Comment (iamsaurav28) [inline Documentation/slider/slider.html]: give relatable title

File: Documentation/slider/slider.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        <li class="list-inline">
+          <a class="links" href="./button.html">Docs</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="d-flex">
+      <div class="left_div">
+        <div class="box1">
+          <p class="content">Components</p>
+          <hr />
```
Comment (iamsaurav28) [inline Documentation/slider/slider.html:31]: there is space between closing tag of <hr/>

**Review by iamsaurav28 — APPROVED** [review]
overall looks good

File: Documentation/slider/slider.html (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply Documentation/slider/slider.html]: done

**Pull request merged**